servers/zts/src/main/java/com/yahoo/athenz/zts/notification/CertFailedRefreshNotificationTask.java

```diff
@@ -25,17 +25,18 @@
 import com.yahoo.athenz.zts.ZTSConsts;
 import com.yahoo.athenz.zts.cert.InstanceCertManager;
 import com.yahoo.athenz.zts.store.DataStore;
+import com.yahoo.rdl.Timestamp;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.sql.Timestamp;
```

Comment:
I also took advantage of the changes to change the Timestamps used in this notification from "java.sql.Timestamp" to "com.yahoo.rdl.Timestamp".